src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -85,6 +85,12 @@
                     "accompanying sequence dictionary (.dict file).")
     public File REFERENCE_SEQUENCE = Defaults.REFERENCE_FASTA;
 
+    @Option(shortName = "WMC", doc = "Warn on missing contig.", optional = true)
+    public boolean WMC = false;
+
+    @Option(doc="When a contig used in the chain is not in the reference, exit with this value instead of 0.")
```

Comment:
I don't think this should be an input option. make it a protected (for testing) static value.